awscli/autocomplete/local/basic.py

```diff
@@ -442,6 +442,8 @@ def _get_prompt_for_list(self, arg_model, parsed_input):
         # - if it's a nested structure we'll get the whole list as an input
         if not isinstance(parsed_input, list):
             parsed_input = [parsed_input]
+        if not len(parsed_input):
+            parsed_input = ['']
```

Comment:
A couple questions here:

1) Could we just do `if not parsed_input` instead? It seems a little strange that we would be basing it off the result of a `len` when that should not be needed.
2) Can we maybe add a comment on how this fixes the issue? It is not really clear what the purpose of these lines are.